src/v2/components/TotalBlocksMeter/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import { propType } from 'graphql-anywhere'
+
+import totalBlocksMeterFragment from 'v2/components/TotalBlocksMeter/fragments/totalBlocksMeter'
+
+import Meter from 'v2/components/UI/Meter'
+import Box from 'v2/components/UI/Box'
+
+export default class TotalBlocksMeter extends Component {
+  static propTypes = {
+    me: propType(totalBlocksMeterFragment).isRequired,
```

Comment:
There's TypeScript support now, why still JS and using PropTypes?

Reply:
Oops sorry, was moving fast here. This is copy-pasta from another component. Will promise to make all future new components TypeScript.

Reply:
👍